recipes/conda-ecosystem-site-packages-isolation/meta.yaml

```
@@ -0,0 +1,28 @@
+package:
+  name: conda-ecosystem-site-packages-isolation
+  version: 0.1.0
+
+build:
+  number: 0
+
+test:
+  requires:
+    - python
+  commands:
+    - python -c "import sys, site; sys.exit(0 if site.ENABLE_USER_SITE==False else 1)"
+
+about:
+  home: http://conda-forge.org/
+  summary: Prevent the environment's Python from searching for packages outside the environment
+  description: >
+    With respect to the list of locations searched by Python for packages, mimic the
+    default behavior of virtual environments created by Python's built-in "venv" module.
+    Python in the conda environment gets isolated from system site-packages, including user
+    site-packages.
```

I'd suggest leading with *what* it does, then mentioning this aligns with `venv` behavior.

```suggestion
    This package uses the mechanism provided by Python's `site` module to ignore 
    system-level site-packages via setting `include-system-site-packages=false` in a 
    prefix-level "pyvenv.cfg". This aligns Conda environments with the default behavior 
    of `venv` virtual environments.
```